parity/params.rs

```diff
@@ -43,6 +43,7 @@ pub enum SpecType {
 	Ellaism,
 	Easthub,
 	Social,
+	EOSC,
```

Comment:
"EOS Classic"

Reply:
Ethereum Classic is using "Ethereum Classic" on their chain spec json, and "Classic" on parity/params.rs file.

It is same with "Ethereum Social" also so I would like to ask using "EOS Classic" on our json file and "EOSC" here 😄